lib/private/normalize-datastore-config/index.js

```diff
@@ -321,6 +321,7 @@ module.exports = function normalizeDatastoreConfig (dsConfig, whitelist, expecte
     }
 
     // If relevant, validate that the RIGHT protocol was found.
+    // TODO: support array of protocols
```

Comment:
why would there be more than one?  Is this because of the atlas thing?

Reply:
@mikermcneil yes. Basically, valid MongoDB protocols are now `mongodb` and `mongodb+srv` https://docs.mongodb.com/manual/reference/connection-string/#dns-seedlist-connection-format